### ISSUE TITLE
Prepaid TWINT & other banks

### ISSUE THREAD

**Issue by ghost** [PrivSec-dev/banking-apps-compat-report#228]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### App name

Prepaid TWINT & other banks

### Link to app

https://play.google.com/store/apps/details?id=ch.twint.payment

### App version

v33.1.0.67

### Country of the app

Switzerland 

### Build Number

TQ1A.230205.002.2023030400

### Device list

Pixel 7

### Profile app tested in

Owner profile

### Google Play installed

Installed

### Google Play services Network permission revoked?

- [ ] Revoked
- [X] Not revoked
- [ ] I did not have Google Play services installed

### Native code debugging

- [X] Enabled
- [ ] Disabled

### Exploit protection compatibility mode

- [ ] Enabled
- [X] Disabled

### Stock OS compatibility

- [X] Works
- [ ] Does not work
- [ ] Not tested

### Description of the app's functionality

Sending and receiving money through your phone number.

### Are there any extra notes you think users should know about?

ONLY WORKS WHEN INSTALLED FROM AURORA. play store does not let install. "This app won't be avalaible for your device".

**Comment by spring-onion** [comment]
@CLDKT Hi there, thanks for reporting. Do you get an error message, a crash or something else?

**Comment by CLDKT** [comment]
Hi. Thanks for following-up. If I search the app in play, it says "This app won't work on your device". So I cannot even download it. What does work are the bank specific apps, like TWINT from UBS or  other banks. Hope that helps somehow.

**Comment by spring-onion** [comment]
@CLDKT Judging by the initial report that's been the case for a while now, have you tried installing it through the aurora store yet?

**Comment by CLDKT** [comment]
Ok tried it. With Aurora, anonymous account, it works. Would it be possible to
make it work via play? Tried Revolut since I was there, still doesn't work.

**Comment by itsmeluciaa** [comment]
As of today (GOS 2026062801) the Prepaid Twint app is not avaliable for me not my friend on the Google Playstore (This app wont work for your device).
I have installed it from Aurora, tho the App does not work; i input the login pin, got loged in. As soon as i did any action after that i was told the app ran into a internet connetion issue.
I reinstalled GOS today to see if it still wasnt avaliable on google play store, wich it wasnt.
Did anyone else run into these issues?
